### PR TITLE
Added Linux question 

### PR DESCRIPTION
**Pull request by tripura-kant**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org). 
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

Added Linux question for [Developer_Quiz_Site](https://github.com/freeCodeCamp/Developer_Quiz_Site)

### CODE REVIEW

**Comment by tripura-kant** [comment]
Added Linux question . Kindly review .

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,14 @@
 const linuxQuiz = [
+  {
+    Question: "In Linux, which command you can use to find memory being used by Linux?",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:3]:
```suggestion
    Question: "Which command finds memory being used by Linux?",
```

File: src/data/linux-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+    Answer: "All of these",
+    Distractor1: "free-m",
+    Distractor2: "htop",
+    Distractor3: "vmstat",
+    Explanation:
+      "The top command is used to display dynamic real-time information. You can use free-m ,htop ,top or vmstat to find memory being used by Linux.",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:9]:
```suggestion
      "The top command is used to display dynamic real-time information. You can use free-m, htop, top or vmstat to find memory being used by Linux.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by tripura-kant** [comment]
Changes are made. Thanks for review.

**Comment by jdwilkin4** [comment]
@tripura-kant 

It looks like you accidentally created a new file. 
Please remove that extra file and only change the existing quiz file

**Comment by tripura-kant** [comment]
Yes, the extra file has been deleted now. Kindly review @jdwilkin4

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**